bitcoin/transaction.py

```diff
@@ -480,3 +481,30 @@ def mksend(*args):
         outputs2 += [{"address": change, "value": isum-osum-fee}]
 
     return mktx(ins, outputs2)
+
+def mk_opreturn(msg, rawtx=None, json=0):
```

Comment:
the json parameter actually needs to be renamed jsonfmt, since `return orhex if not json else orjson` will never return orhex (since json is the name of the module)
See my fork: https://github.com/simcity4242/pybitcointools/